tests/test_utils.py

```diff
@@ -666,3 +666,66 @@ def walk(s, recurse):
 
     schema = _WalkCoreSchema().handle_call_schema(schema, walk)
     assert schema['return_schema'] == {'type': 'int'}
+
+
+def test_pretty_print(capfd):
+    # Included metadata
+    schema = core_schema.AnySchema(
+        type='any',
+        ref='meta_schema',
+        metadata={'schema_type': 'any', 'test_id': '42'},
+        serialization=core_schema.simple_ser_schema('bool'),
+    )
+    expected_meta_info = "{\n    'type': 'any',\n    'ref': 'meta_schema',\n    'metadata': {'schema_type': 'any', 'test_id': '42'},\n    'serialization': {'type': 'bool'}\n}\n"
```

Comment:
Maybe multiline strings could be used?

Reply:
@Viicos, yep, I think we're just waiting on formatting changes, but otherwise looks good!

Reply:
Thanks for the suggestions! I'll go through and get the tests passing in the workflow later today and then re-request review.

Reply:
Changes made and checks passed @sydney-runkle